src/components/POLine/Vendor/VendorForm.test.js

```diff
@@ -5,6 +5,10 @@ import { render, screen } from '@folio/jest-config-stripes/testing-library/react
 import { arrayMutators } from 'fixtures/arrayMutatorsMock';
 import VendorForm from './VendorForm';
 
+jest.mock('../../../common/IfFieldVisible', () => ({
+  IfFieldVisible: jest.fn(({ children }) => children),
```

Comment:
I guess there is no need to use `jest.fn()` if you are not going to test this component, just mock it as `IfFieldVisible: ({ children }) => children,`